src/test/java/exploratory/decodes/ConcurrentQueueTest.java

```diff
@@ -1,10 +1,9 @@
-package decodes.test;
+package exploratory.decodes;
 
 import java.util.Iterator;
 import java.util.concurrent.ConcurrentLinkedQueue;
 
-public class ConcurrentQueueTest
-	implements Runnable
+class ConcurrentQueueTest
```

Comment:
Looks like this is an exploratory test of the Java `Queue` data structure. This looks like a candidate for deletion unless someone thinks it is useful to keep around. It doesn't test any OpenDCS functionality or concurrent Java functionality as everything is run sequentially.

Reply:
yeah, remove it.